### ISSUE TITLE
Rename "Mavlink" stuffs to that of "MAVLink"

### ISSUE THREAD

**Issue by shakthi-prashanth-m** [dronecore/DroneCore#329]
These classes use lower case for MavLink:
```
class MavlinkCommands {...};
class MavlinkParameters {...};
class MavlinkReceiver {...};
class MavlinkChannels {...};
```
In QGC source and elsewhere the spelling is MAVLink. Should I change these to use the correct case (e.g. `MavlinkCommands` to `MAVLinkCommands`, etc.)

Please suggest: @hamishwillee @julianoes @JonasVautherin .
Thanks.

**Comment by hamishwillee** [comment]
If MAVLink is always implied then it isn't needed. If something is intended to be generic then it shouldn't be prefixed. You'd use a MAVLink where something must always be for working with MAVLink and only makes sense in that context.

**Comment by shakthi-prashanth-m** [comment]
OK. Thanks for rephrasing my question @hamishwillee . :)

**Comment by shakthi-prashanth-m** [comment]
Addressed as part of #310.

**Comment by julianoes** [comment]
To me mavlink is sort of like a word which is why I wrote it that way. We can change it but I don't know if the churn is really warranted.

**Comment by hamishwillee** [comment]
Probably not, but already done :-)

**Comment by shakthi-prashanth-m** [comment]
Addressed in #310 .
